app/test_split_integer.py

```diff
@@ -2,20 +2,28 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    goal = split_integer(32, 6)
+
+    assert sum(goal) == 32
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    goal = split_integer(30, 6)
+
+    assert all(True if num == 5 else False for num in goal)
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    goal = split_integer(32, 1)
+    assert goal[0] == 32
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    goal = split_integer(32, 6)
+    assert all(True if goal[i] == sorted(goal)[i]
+               else False for i in range(len(goal)))
```

Comment:
Do you really need this complex logic? You can use some concrete example to make test easy to understand.
for example `assert goal == [1,1,2,2]`